content/en/docs/about/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
 title = "About"
-description = "Information about Kubeflow software, community, docs, and events."
+description = "About Kubeflow and its community"
```

Comment:
After this PR's refactoring, I believe this doc area has become `About Kubeflow community` than `About Kubeflow and its community`. The Kubeflow introduction has been moved to `Getting Started` doc area.

Reply:
I would still say its "information about Kubeflow", (e.g. the docs style guide, and contributing)

Reply:
I see, we can discuss about the description separately. Let's get the doc restructuring in this PR merged in first.